### ISSUE TITLE
[IMPORTANT] YouTube is banning accounts used for age-gate bypassing in large instances

### ISSUE THREAD

**Issue by kuylar** [lighttube-org/LightTube#154]
# TL;DR

YouTube started banning accounts that load a lot of videos from watching more videos. If you used your main Google account for the LightTube configuration, please change/remove it before its too late

# More details

YouTube has started banning accounts that load a lot of videos. The first report we've seen so far is from 01 July 2024, at roughly 7 AM (will be updated if earlier reports are found). 

Banned accounts will not be able to watch YouTube videos, and will be met with a "Video Unavailable" screen, visible below  
![Big Buck Bunny's watch page on YouTube showing the "Video Unavailable" error](https://github.com/lighttube-org/LightTube/assets/52961639/8b491fd7-444d-4c4d-a164-e9fc36fe4ef6)  

This ban seems to only affect video playback, as other features such as searching, browsing, making comments, liking videos seem to work fine.

This issue will be updated with more information if there is any. For now, it's recommended to remove your Google account from your LightTube configuration if you care about that account.

**Comment by kuylar** [comment]
Update/Theory:

Just checked zerodytrash's Simple-YouTube-Age-Restriction-Bypass extension and its account proxy server, and it seems like they werent affected by this ban wave. Looking into the account proxy code, they send *way* more data than LightTube does. Gonna update nightly soon with another account & more stuff in the client context

**Comment by kuylar** [comment]
The theoretical fix above is up in tube-nightly.kuylar.dev, with another account set for age-gate bypassing. Would love some traffic to see if we get banned again.

If everything fails, I'm just gonna remove authentication altogether, and just use the TVHTML5_EMBEDDED client.

**Comment by kuylar** [comment]
Update:

Seems like banned accounts can still use the `IOS` client to get videos. LightTube already used the `IOS` client, however, since it lacks the muxed formats (itag 18 & 22) I don't think we can switch to those.

**Comment by rjdg14** [comment]
As someone who uses FreeTube and NewPipe because I prefer them over the default YouTube interface (which has become terrible in recent years, though I choose to have a Premium subscription) I'm a little concerned about these recent changes. I've been having no issues myself but some people have said that YouTube has been blocking video playback to IP addresses that it detects to be using yt-dlp (which I don't use, though I think FreeTube contains some components of it). I'm unsure whey are blocking the IPs of low level users of it or if they are blocking the addresses with the most significant use (eg: accessing a significant number of videos, which I think can be achieved with yt-dlp and can be indicative of bot behaviour/crawling).

Are the IP/account blocks that people have been experiencing in recent weeks believed to be indefinite/permanent or temporary?

**Comment by kuylar** [comment]
So far they're permanent, and support cases for getting unbanned seem inconclusive. It's best to use burner/throwaway accounts

**Comment by nikooo777** [comment]
it's been about a month, question for those affected by the ban: were you unbanned or are you still banned after all this time?

**Comment by ifsdk** [comment]
yesterday i checked it works again

**Comment by ifsdk** [comment]
again getting "This content isn’t available."

**Comment by peter-joo** [comment]
it works for me now, my estimation is that I had a six weeks ban.
